src/com/hms/flexyosisoftconnector/OSIsoftServer.java

```diff
@@ -272,21 +272,39 @@ public int initTags() throws JSONException {
         // setup type
         String messageTypeHeader = "&messagetype=type";
 
-        String payload = PayloadBuilder.getTypeBody();
         res =
             RequestHTTPS(
                 OSIsoftConfig.getOmfUrl(),
                 "Post",
                 OSIsoftConfig.getOmfPostHeaders() + messageTypeHeader,
-                payload,
+                PayloadBuilder.getStringTypeBody(),
+                "",
+                responseFilename);
+
+        res =

```

Comment:
is there a reason for storing the return value of RequestHTTPS if you are not checking the value?

Reply:
yeah my bad, I was at one point. there really is not

Reply:
fixed this and actually checked the codes